src/css/custom.css

```diff
@@ -241,3 +241,24 @@ td {
     display: block !important;
   }
 }
+
+.tabs {
+  padding-left: 0;
+}
+
+.tabs__item {
```

Comment:
Okay one more thing I noticed on this - the left/right padding is not the same, for some reason. We should explicitly set it.
```
padding-left: 12px;
padding-right: 12px;
```